netflix/src/redux/films.ts

```diff
@@ -0,0 +1,23 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+export const filmSlice = createSlice({
+  name: "film",
+  initialState: {
+    films: [],
+    results: 0,
```

Comment:
Why do you need to store number of films as separate state item? This breaks `single source of truth` principle. Instead you should have `selector` that returns `state.fimls.length`.